packages/util/src/code-block.ts

```diff
@@ -0,0 +1,25 @@
+import { zip } from './array';
+import { trim, trimRight } from './string';
+
+export const codeBlock = (strings, ...args) => {
```

Comment:
doc missing or `hidden` also I do not see where this is used.

Reply:
Also here stronger typeing for arguments.

Reply:
I will add the docs and types. It is used in the string templates:
```
codeblock`bla`
```

Reply:
Done.